standalone/instructlab-rhoai.md

```diff
@@ -0,0 +1,320 @@
+# Distributed InstructLab Training on RHOAI
+
+This file documents an experimental install of InstructLab on Red Hat OpenShift AI.
+
+## Pre-requisites
+
+* An OpenShift cluster with
+    * Sufficient GPUs available for training.
+        * 4x NVIDIA A100 GPU
+    * Red Hat - Authorino installed
+    * Red Hat Openshift Serverless installed
+* An OpenShift AI installation, with the Training Operator and kserve components set to `Managed`
+    * A data science project/namespace
+* A [StorageClass](https://kubernetes.io/docs/concepts/storage/storage-classes/) that supports dynamic provisioning with [ReadWriteMany](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes) access mode (see step 3 below).
+* An AWS S3 compatible object store. Alternative object storage solutions such as Ceph, Nooba and MinIO are also compatible.
+* SDG data generated that has been uploaded to an object store (see step 1 below).
```

Comment:
Maybe add the `oc` binary should be installed locally.

Reply:
Is this draft seems correct? 
```
The `oc` command line tool binary installed locally to create and manage kubernetes resources.
```